sktime/markov/_base.py

```diff
@@ -225,8 +226,9 @@ def score_cv(estimator: _MSMBaseEstimator, dtrajs, lagtime, n=10, count_mode="sl
 
     Parameters
     ----------
-    estimator : MSMBaseEstimator like
-        estimator to produce models for CV.
+    fit_fetch : callable, optional, default=None
```

Comment:
It would be nice to provide an example of this in the code docs like

```
Examples
----------

[whatever]
```

Reply:
also where is the part where the default is assigned to `None`? maybe i don't understand the typing

Reply:
yeah examples for docs are on my todo list and will follow in a separate PR. also it is not optional so the default=None is wrong, good catch :)